### DISCUSSION TITLE
Vue Support

### DISCUSSION THREAD

**Discussion by soullpoint** [remirror/remirror#438] [Ideas] [answered]
## Description

You mention that this is intended to be a cross platform solution, with Angular support to come; I was wondering if there are any plans to support Vue?

In either case, would you happen to have any documentation / be able to provide some direction for someone who would like to use your library in a Vue based project? 

Thanks!

**Comment by ifiokjr** [comment, marked as answer]
[`tiptap`](https://github.com/ueberdosis/tiptap) is currently doing a fantastic job of supporting Vue and I think they have some exciting things planned in their upcoming release.

**Comment by soullpoint** [comment]
@ifiokjr Apologies for the late response, I couldn't find the issue after it was moved into the discussions.

Thanks for the link, it definitely looks like an interesting project. I will have a play with it. However, from a brief look around, it looks like they don't support Typescript as of yet. They're also (understandably) looking for funding before they can commit to a v2 which might be written in Typescript - though they're not 100% convinced of that yet either.

Based on this, I may end up coming back here. If so, I'll try and get something running based on the React module.

Thanks again.

**Reply by ifiokjr** [reply]
> I couldn't find the issue after it was moved into the discussions.

Oh that's interesting, I was hoping it would notify you. And yes, discussions doesn't current have the visibility that issues have, I think that will be improved.

I'm glad you could take a look at `tiptap`. If you do decide that it's worth adding a `vue` module, please do connect with us on the [discord channel](https://discord.gg/C4cfrMK). I would be interested in helping out where possible.

**Comment by GitHubish** [comment]
@ifiokjr Does ReMirror still aim to be compatible with other frameworks than React? I'm currently using Tiptap with Angular, but I'll be curious to study the differences with Remirror in case it becomes usable with Angular
